### DISCUSSION TITLE
请问在输入数字选项操作的时候，输入空格和1或者1加上空格算不算输入错误？还是不会提供这种测试样例？

### DISCUSSION THREAD

**Discussion by tang-mouren** [liusy58/NJU_AdvancedProgramming_SP22#10] [Q&A] [answered]
使用了strcmp函数会导致空格1的输入非法，不知道这会不会影响样例的通过。

**Comment by Price1999a** [comment, marked as answer]
你可以实现一个类似于`trim`函数的功能

另外，你完全可以将这样的输入视作**非法**

**Reply by liusy58** [reply]
最好可以摒弃C语言中`char*`字符串，使用C++所有的`std::string`来减轻你的痛苦。毕竟这是一门C++的课程。

**Reply by tang-mouren** [reply]
好的

**Reply by Price1999a** [reply]
[`QString`](https://doc.qt.io/qt-5/qstring.html) 才是真正的未来

**Reply by RicoloveFeng** [reply]
> [`QString`](https://doc.qt.io/qt-5/qstring.html) 才是真正的未来

这次实验不能用 Qt 的东西（

**Comment by RicoloveFeng** [comment]
Try `std::stoi`
